.github/workflows/ci.yml

```diff
@@ -91,6 +91,42 @@ jobs:
           start: npm run dev:standalone
           wait-on: "http://localhost:3000/search"
 
+  api-build:
+            name: Build API
+            runs-on: ubuntu-latest
+
+            env:
+              api-directory: ./api
+
+            strategy:
+                matrix:
+                  dotnet-version: [ '9.0.x' ]
+
+            defaults:
+              run:
+                working-directory: ${{ env.api-directory }}
+
+            steps:
+              - name: Checkout source
+                uses: actions/checkout@v4
+
+              - name: Setup dotnet ${{ matrix.dotnet-version }}
+                uses: actions/setup-dotnet@v3
```

Comment:
Looks like there's a v4 of this action. Any reason not to use that?

Reply:
No - updated to V4.